### ISSUE TITLE
Order confirmation diplays wrong informations

### ISSUE THREAD

**Issue by Cecilia-Hn** [openfoodfoundation/openfoodnetwork#8762]
## Description

On the 1st January, on the 922 hub, a client ordered vegetables (including one onion) for 23.60 euros.
The order number is R652142303.

Few minutes later, the client received his confirmation order by mail for 23.60 euros, while the 922 hub was receiving confirmation order for 71.20 euros (with 20 onions more).
It seems that the client didn't changed his order (I checked sent mail on sendgrid).

The order registered in OFN is the one for 21 onions and 71.20 euros.

**This is the order registered in OFN and received by the Hub :** 

![Capture OL3](https://user-images.githubusercontent.com/67476305/150102533-d77f64d1-4108-46e5-8234-ae908f30a9ef.PNG)

![Capture OL](https://user-images.githubusercontent.com/67476305/150101168-1efae66e-c312-45e8-b060-8425447609f9.PNG)

**This is the confirmation order received by the client :** 

![Capture OL2](https://user-images.githubusercontent.com/67476305/150102015-a7feed36-6753-4da3-ae02-69f45d5bd6ba.PNG)

The client received 20 onions more than ordered.

## Workaround
The client had to show confirmation order to the hub and the hub had to change the quantity of product delivered.

**Comment by lin-d-hop** [comment]
Hi @Cecilia-Hn 
Do you also have the order confirmation email received by the Hub? That would be nice to also cross reference.
Thanks.

**Comment by RachL** [comment]
> It seems that the client didn't changed his order (I checked sent mail on sendgrid).

@Cecilia-Hn note that if the customer edit their order after confirmation, OFN does not re-sent an email. I think to rule out user error, we should ask for a screenshot of the customer account. If there is 21 there too, someone definitely edited the order.

On metabase I can see that the order was updated after completion:

![image](https://user-images.githubusercontent.com/6525576/150112738-0f9949cd-d052-4cde-8c5a-8d9a5f1a312a.png)

But I don't know if it was the user or the hub.

@openfoodfoundation/core-devs do we have a way to know who did it?

**Comment by mkllnk** [comment]
> On metabase I can see that the order was updated after completion
> do we have a way to know who did it?

No, we don't. And there is the unlikely possibility that our code changed the order. Does the updated_at time correlate with any of the emails?

**Comment by RachL** [comment]
@mkllnk emails are sent on completed_at time not on update_at time...

@Cecilia-Hn when you say:
> the client received his confirmation order by mail for 23.60 euros, while the 922 hub was receiving confirmation order for 71.20 euros (with 20 onions more).

You mean that the hub **email** says 71.20 euros? 

Are both email sent at the same time?

**Comment by Cecilia-Hn** [comment]
#You mean that the hub email says 71.20 euros?

Yes, that what is described by the hub

#Are both email sent at the same time? 

I let you know as soon as I got the information

**Comment by mkllnk** [comment]
I can see from the screenshot that the customer email was sent at 11:28. When was shop's email sent?

Both emails are scheduled to be sent at the same time, ten seconds after the order was placed. The email sending is done in a background job though and we can't exactly tell when they are sent.

**Comment by mkllnk** [comment]
I noticed though that they do allow order changes. So it's possible that the customer changed it, maybe even by mistake. Sometime my cat jumps on my keyboard... :cat:

**Comment by mkllnk** [comment]
I looked through the access lock:

- Adding to cart from 11:15, several items, only one onion.
- Going to checkout.
- Login.
- Checking out at 11:25.
- Seeing the order confirmation.
- No further action by the customer.
- Looking at the order again at 17:49, probably to check the quantity.

Our background process log says that two emails were sent at 11:25, in parallel, within the same second. And there doesn't seem to be any other action on the website in between. That's very weird. No user action. So it looks like our software did that but I don't know how two emails of the same order at the same time give different results.

**Comment by mkllnk** [comment]
Okay, I now realised that we don't actually have proof of the customer's email saying 21 onions. As Rachel pointed out, the order was changed the next evening at 22:07 and I can see in the logs that the front end was used for that. I can also see the login five minutes earlier. It was the customer who changed the order. It may have been a mistake but technically, it was changed by the customer on Sunday, 2nd of January 2022 at 22:07 French time.

Closing.
